### DISCUSSION TITLE
Azure AD replacing common with tenant ID not getting saved

### DISCUSSION THREAD

**Discussion by hkjarral** [goauthentik/authentik#7600] [Q&A]
![Xnip2023-11-16_14-21-03](https://github.com/goauthentik/authentik/assets/42726887/bd2876b4-245f-4d5d-97da-dc6724347586)
![Xnip2023-11-16_14-21-44](https://github.com/goauthentik/authentik/assets/42726887/199a8742-70cb-4e3c-b76e-af6d10cbd93f)

Everytime I save it, it reverts back to common.

**Comment by BeryJu** [comment]
This will be fixed with https://github.com/goauthentik/authentik/pull/7560 in 2023.10.4
